index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="portfolio.css">
+    <title>Nsanzimfura portfolio</title>
+
+</head>
+<body>
+    <header>
+        <nav class="top-navigation-bar">
+            <h2 class="header-welcome">Welcome</h2>
+            <img class="menu-icon" src="./assets/header-files/icon-left-side-menu.svg" alt="menu icon" />
+        </nav>
+    </header>
+    <main>
+        <section class="main-page">
+            <p class="introduction-text"><span class="dark-blue">Hey there. I'm <br>Eden<br></span>
+            <span class="green-content">I'm a software developer</span></p>
```

Comment:
- [ ] Instead wrapping this sub-title with a `p` tag, kindly change it to an `h3`, since you are using the `h2` on your logo.